infrastructure/Taskfile.yml

```diff
@@ -1066,6 +1066,18 @@ tasks:
             | grep -v ": 0 matches"
           echo "done"
 
+    sites:latest-backup:
+      desc: Show the timestamp for the last successful backup for each site
+      vars:
+        SITES:
+          sh: lagoon list projects --output-json | jq -r ".data[].projectname"
+      cmds:
+        - for: { var: SITES }
```

Comment:
This is a new approach to running a task across all projects using [Taskfile loops](https://taskfile.dev/usage/#looping-over-values).

I think this reads a little better than our current approach with bash.

Reply:
👍 we use it elsewhere, and it makes a lot of sense here